benchopt/cli/helpers.py

```diff
@@ -33,15 +33,27 @@
 )
 @click.argument('benchmark', default=Path.cwd(), type=click.Path(exists=True),
                 shell_complete=complete_benchmarks)
-def clean(benchmark, token=None, filename=None):
+@click.option("--filename", "-f", "filename",
+              type=str, default="all",
+              help="Name of the output file to remove.")
+def clean(benchmark, token=None, filename='all'):
 
     benchmark = Benchmark(benchmark)
-
     # Delete result files
     output_folder = benchmark.get_output_folder()
-    if output_folder.exists():
+    if output_folder.exists() and filename == 'all':
         print(f"rm -rf {output_folder}")
         rm_folder(output_folder)
+    else:
+        for ext in [".csv", ".html"]:
+            if ext == ".html":
+                to_remove = output_folder / f"{benchmark.name}_{filename}"
+            else:
+                to_remove = output_folder / filename
+            file = to_remove.with_suffix(ext)
+            if file.exists():
+                print(f"rm {file}")
+                file.unlink()
```

Comment:
Also need to remove it from the benchmark html result page and the cache_list?

Reply:
I removed the entry associated to the remove files in the cache.
Using `benchopt plot` this removes the entry in the index html page too.